mlxtend/feature_selection/sequential_feature_selector.py

```diff
@@ -14,25 +14,73 @@
 import sys
 from copy import deepcopy
 from itertools import combinations
-from sklearn.metrics import get_scorer
+from sklearn.metrics import get_scorer, roc_auc_score, accuracy_score
```

Comment:
I don't think we need to import roc_auc_score and accuracy_score. 

Sorry, I think this is completely my fault. I didn't make the arguments in the functions I used in my example notebook generic enough.

I have tried clarified my thoughts regarding the scoring metric below.